doc/nrf/ug_nrf9160.rst

```diff
@@ -121,28 +117,23 @@ Delta patches
 Band lock
 *********
 
-The band lock is a functionality of the application that lets you send an
-AT command to the modem instructing it to operate only on specific bands.
-Band lock is handled by the **LTE Link Control** driver and is by default
-disabled in its Kconfig file.
+The band lock is a functionality of the application that lets you send an AT command to the modem instructing it to operate only on specific bands.
+The band lock is handled by the **LTE Link Control** driver.
```

Comment:
This kind of formatting is reserved only for PCB elements. Please use emphasis formatting or no formatting.

Reply:
right, fixed